mods/key_value.js

```diff
@@ -250,11 +254,18 @@ KVBucket.prototype.putRevision = function(restbase, req) {
                 key: rp.key,
                 tid: tid,
                 value: req.body,
-                'content-type': req.headers['content-type']
-                // TODO: include other data!
+                'content-type': req.headers['content-type'],
+                headers: req.headers
             }
         }
     };
+    if (req.headers && req.headers['max-age']) {
```

Comment:
We can't sanely support per-row TTLs in the backend, so will probably have to replace this with a shared stash bucket with a retention policy.